app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command, file, direct = command.split(" ")
```

Comment:
first check if the length of the split result is three, otherwise it will throw an error